### PR TITLE
Proxy Apps with icon to distinguish from native apps

### PR DESCRIPTION
**Pull request by romer8**
- [x] New field added in the ProxyApps model called display_external
- [x] Conditional rendering in case the display_external of the proxy app is disable
- [x] Migration created
- [x] Documentation updated

Icon appearance:
image

Admin appearance:
image

### CODE REVIEW

**Comment by sdc50** [comment]
@romer8 I like this approach. I think it may be more clear to have the checkbox label say `Display External Indicator`. I'm also curious how it looks if the app name is longer. How would it look if the icon were about the same size as the `info` icon that shows up at the top of the app card?

File: docs/tethys_portal/admin_pages.rst [hunk 1]
```diff
@@ -279,6 +279,7 @@ Proxy apps are links to external web applications or websites. They are represen
 * **Show in apps library**: Display the app on the Apps Library page. The app is still accessible by URL, but no app tile will be shown in the apps library.
 * **Open in new tab**: Open the proxy app link in a new tab or window when on. Open it the same tab or window when off.
 * **Order**: Force the order that apps are listed on the Apps Library page. Default order is alphabetical.
+* **External Display**: By default, this feature is turned off. It adds an icon to distinguish proxy apps from regular apps.
```
Comment (swainn) [inline docs/tethys_portal/admin_pages.rst:282]:
```suggestion
* **Display External Icon**: By default, this feature is turned off. When enabled, it adds an icon to the app tile to distinguish the proxy app from regular apps, indicating that it is an external link.
```

File: tethys_apps/models.py [hunk 2]
```diff
@@ -1276,6 +1276,7 @@ class ProxyApp(models.Model):
     show_in_apps_library = models.BooleanField(default=True)
     open_in_new_tab = models.BooleanField(default=True)
     order = models.IntegerField(default=0)
+    display_external = models.BooleanField(default=False)
```
Comment (swainn) [inline tethys_apps/models.py:1279]:
```suggestion
    display_external_icon = models.BooleanField(default=False)
```

**Review by swainn — COMMENTED** [review]
The icon you provided is the right symbol, but too big and the circle around it makes it look off-center b/c of the shape of the icon. I recommend we use the bootstrap icon with no circle around it: https://icons.getbootstrap.com/icons/box-arrow-up-right/. Make it a little smaller too.

I like the placement of the icon in the bottom right corner of the app tile, but please add white space to offset it from the corners a bit.

File: tethys_apps/models.py (hunk 2 above)
Comment (romer8) [reply tethys_apps/models.py:1279]: This was changed as suggested

File: docs/tethys_portal/admin_pages.rst (hunk 1 above)
Comment (romer8) [reply docs/tethys_portal/admin_pages.rst:282]: This was changed as suggested

**Comment by romer8** [comment]
@swainn @sdc50 I have changed the style of the icon to look like this:

image

I have made the name long as well to see if it looks good.
The name has been changed in the model and admin page also to look like **Display External Icon**

**Comment by swainn** [comment]
> I have made the name long as well to see if it looks good. The name has been changed in the model and admin page also to look like **Display External Icon**

Excellent. I like this version a lot more. I'd like to get feedback from @sdc50 and/or @shawncrawley on this too.

**Review by swainn — APPROVED** [review]

**Review by shawncrawley — APPROVED** [review]
Yeah, I like this. Pretty straightforward and looks good.

**Pull request merged**